### PR TITLE
Restrict domain name length for Inventek module's DNS lookup function

### PR DESCRIPTION
**Pull request by aggarg**
Description
-----------

The inventek WiFi module supports maximum domain name length of 64. Trying to resolve a longer name leaves the module in a non-responsive state. This PR adds a check to prevent calling `ES_WIFI_DNS_LookUp` for longer names, thereby preventing the module from going into a non-responsive state.

The following is from the datasheet of the module:
```
'D0' DNS Lookup:
This command performs a DNS lookup of a Domain Name to get its IPv4 address. The Domain Name is limited to 64 characters.
```

This PR also increases the stack size of test runner task to address a stack overflow.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c [hunk 1]
```diff
@@ -59,7 +59,7 @@
 #define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 4 )
 #define mainLOGGING_MESSAGE_QUEUE_LENGTH    ( 15 )
 
-#define mainTEST_RUNNER_TASK_STACK_SIZE     ( configMINIMAL_STACK_SIZE * 8 )
+#define mainTEST_RUNNER_TASK_STACK_SIZE     ( configMINIMAL_STACK_SIZE * 10 )
```
Comment (abhidixi11) [inline vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c:62]: Do we know stack overflow was happening ?

**Review by abhidixi11 — APPROVED** [review]

File: vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c (hunk 1 above)
Comment (aggarg) [reply vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c:62]: After the fix, the module was no-longer unresponsive for the rest of the tests. When those tests ran, the test runner task was abruptly getting deleted. The reason was a return address getting corrupted on the stack because of stack overflow.

**Review by paulbartell — APPROVED** [review]
